src/test/java/uk/gov/hmcts/reform/managecase/service/noc/PrepareNoCServiceTest.java

```diff
@@ -0,0 +1,366 @@
+            mockUserCaseRolesForSolicitor();
+
+            Map<String, JsonNode> caseData = new HashMap<>();
+            caseData.put(COR_FIELD_NAME, changeOrganisationRequestToJsonNode(changeOrganisationRequest()));
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, JURISDICTION, "", CASE_TYPE, caseData,
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("No OrganisationPolicy found on the case data.")
+            );
+        }
+
+        @Test
+        @DisplayName("should error when Solicitor is not recorded on an OrganisationPolicy on the case")
+        void shouldErrorWhenSolicitorIsNotRecordedInAnOrganisationPolicyOnTheCase() {
+            mockUserCaseRolesForSolicitor();
+            mockOrgIdentifierInPrdFindUsersByOrganisation("orgNotInPolicy");
+
+            Map<String, JsonNode> caseData = new HashMap<>();
+            orgPolicies.forEach((key, value) -> caseData.put(key, organisationPolicyToJsonNode(value)));
+            caseData.put(COR_FIELD_NAME, changeOrganisationRequestToJsonNode(changeOrganisationRequest()));
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, JURISDICTION, "", CASE_TYPE, caseData,
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("The Organisation of the solicitor is not recorded in an Org policy on the case.")
+            );
+        }
+
+        @Test
+        @DisplayName("should error when there is ongoing NoCRequest")
+        void shouldErrorWhenThereIsOngoingNoCRequest() {
+            mockUserCaseRolesForSolicitor();
+
+            Map<String, JsonNode> caseData = new HashMap<>();
+            orgPolicies.forEach((key, value) -> caseData.put(key, organisationPolicyToJsonNode(value)));
+            caseData.put(COR_FIELD_NAME, changeOrganisationRequestToJsonNode(ongoingChangeOrganisationRequest()));
+
+            CaseDetails caseDetails = new CaseDetails(REFERENCE, JURISDICTION, "", CASE_TYPE, caseData,
+                                                      securityClassification, dataClassification);
+
+            ValidationException exception = assertThrows(
+                ValidationException.class,
+                () -> prepareNoCService.prepareNoCRequest(caseDetails)
+            );
+            assertThat(
+                exception.getMessage(),
+                is("There is an ongoing NoCRequest.")
```

Comment:
If string should match validation error then import and match on constant defined in *src/main/java/uk/gov/hmcts/reform/managecase/api/errorhandling/ValidationError.java*.